plugins/jira-dashboard-backend/src/config.ts

```diff
@@ -25,7 +25,9 @@ export function resolveUserEmailSuffix(config: Config): string {
   try {
     return config.getString(JIRA_USER_CONFIG_EMAIL_SUFFIX);
```

Comment:
If this should be optional then the complete try/catch can be removed and replace with 

`config.getOptionalString(JIRA_USER_CONFIG_EMAIL_SUFFIX) || ''`